OWNERS_ALIASES

```diff
@@ -3,11 +3,13 @@
 
 aliases:
   machine-controller-maintainers:
+    - yaa110
```

Comment:
But then why not adding Navid in the sig cluster management group

Reply:
Is he part of sig-cluster-management?

Reply:
He is [not a member ](https://github.com/kubermatic/access/blob/main/github/kubermatic.yaml#L530-L544)of that SIG and our automation will simply remove him from the list when it syncs the owners again. Since it's source of truth is https://github.com/kubermatic/access/blob/main/github/kubermatic.yaml#L530-L544

Reply:
@yaa110 Can you please add your handle where Waleed is pointing. Thanks

Reply:
done: https://github.com/kubermatic/access/pull/1292

Reply:
I can't do much about this PR. If you want to join sig-cluster-management actively. Please take it up in the slack channel with their SIG chair etc ;)

Reply:
Updated and now we have sig-cluster-management updated with Navid.